Source/WebCore/Modules/applicationmanifest/ApplicationManifestParser.cpp

```diff
@@ -73,12 +73,21 @@ ApplicationManifest ApplicationManifestParser::parseManifest(const String& text,
 
     ApplicationManifest parsedManifest;
 
+    parsedManifest.rawJSON = text;
+    parsedManifest.manifestURL = manifestURL;
     parsedManifest.startURL = parseStartURL(*manifest, documentURL);
     parsedManifest.display = parseDisplay(*manifest);
     parsedManifest.name = parseName(*manifest);
     parsedManifest.description = parseDescription(*manifest);
     parsedManifest.shortName = parseShortName(*manifest);
-    parsedManifest.scope = parseScope(*manifest, documentURL, parsedManifest.startURL);
+    auto parsedScope = parseScope(*manifest, documentURL, parsedManifest.startURL);
+    if (parsedScope)
+        parsedManifest.scope = parsedScope.value();
+    else {
```

Comment:
Note that `isDefaultScope` was left uninitialized in the `if` case.

Reply:
I've added a default initializer for `isDefaultScope` as you've suggested, so this code should be fine now.